src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -2125,4 +2125,31 @@ From there, simply click on one of the supported files in the File Explorer and
   <data name="FancyZones_DisableRoundCornersOnWindowSnap.Content" xml:space="preserve">
     <value>Disable round corners when window is snapped</value>
   </data>
-</root>
+  <data name="PowerLauncher_SearchQueryTuningEnabled.Description" xml:space="preserve">
+    <value>Options to fine tune how the results are shown</value>
+  </data>
+  <data name="PowerLauncher_SearchQueryTuningEnabled.Header" xml:space="preserve">
+    <value>Results display tuning</value>
+  </data>
+  <data name="PowerLauncher_SearchClickedItemWeight.Description" xml:space="preserve">
+    <value>Use a higher number to get clicked results to rise faster. The default is 5, 0 to disable.</value>
+  </data>
+  <data name="PowerLauncher_SearchClickedItemWeight.Header" xml:space="preserve">
+    <value>Clicked item weight</value>
+  </data>
+  <data name="PowerLauncher_WaitForSlowResults.Description" xml:space="preserve">
+    <value>Selecting this can help preselect the top, more relevent result, but at the risk of jumpiness</value>
+  </data>
+  <data name="PowerLauncher_WaitForSlowResults.Header" xml:space="preserve">
+    <value>Wait on async results before selecting</value>
+  </data>
+  <data name="PowerLauncher_PluginWeightBoost.Content" xml:space="preserve">
```

Comment:
Is this needed?

Reply:
OK sorry, no we don't need that, will kill.